### PR TITLE
Ana/fix navigation duplicated error

### LINKED ISSUE #4580: Error importing account: NavigationDuplicated
**Describe the bug**

This is the cause of failing e2e tests.

**To Reproduce**

1. Open latest develop with insecure flag
2. Click on Recover with backup code
3. Select a network
4. You will be redirected to validators with the console error: `Navigating to current location ("/polkadot/validators") is not allowed`

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser: 
 - Version: 

**Additional context**

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4580 

**Description:**

Still getting an Uncaught Promise coming from `setNetwork`. Will fix it now

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f1aa3896ef80aea7ef47f87--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
:thinking: the `Uncaught promise` one seems to be more difficult to debug :shrug: tried with try-catch blocks in both routes.js and setNetwork.js but nothing. It is coming from line 53 in setNetwork.js but maybe it is an inner thing

**Comment by faboweb** [comment]
You know about this one?
image

File: app/src/components/network/NetworkList.vue [hunk 1]
```diff
@@ -40,23 +40,12 @@ export default {
   },
   computed: {
     ...mapGetters({ networkId: `network` }),
-    whichFlow() {
```
Comment (faboweb) [inline app/src/components/network/NetworkList.vue:43]: this is not needed anymore? is this handled by the router? does the network list still work in the account creation flow?

Comment (Bitcoinera) [reply app/src/components/network/NetworkList.vue:43]: yes :shrug: everything works just fine. I mean, what is supposed to happen when you click on a NetworkItem? You are either taken to validators or to portfolio, right?

**Comment by Bitcoinera** [comment]
> You know about this one?

I actually do :sweat_smile: Yeah, let me check with this one

File: app/src/components/network/NetworkList.vue (hunk 1 above)
Comment (faboweb) [reply app/src/components/network/NetworkList.vue:43]: we use the same component in the sign in flow. so please check if removing this breaks the sign in flow.

Comment (Bitcoinera) [reply app/src/components/network/NetworkList.vue:43]: oops, ok, will check it

**Comment by Bitcoinera** [comment]
I didn't check that yet. I think it was broken in fact, in sign in flow. Please don't be so impatient :sweat_smile:

**Comment by Bitcoinera** [comment]
Actually we broke extension without noticing. Opening PR

**Pull request merged**